dist/assembly.xml

```diff
@@ -161,6 +161,30 @@
       <useTransitiveFiltering>false</useTransitiveFiltering>
     </dependencySet>
 
+    <!-- Cassandra module -->
+    <dependencySet>
+      <outputDirectory>${hawkular.dist.zip.root.dir}/modules/org/apache/cassandra/main</outputDirectory>
+      <includes>
+        <include>org.apache.cassandra:cassandra-all:jar</include>
```

Comment:
That contains Cassandra, but it's not started as embedded. The embedded EAR still bundles its own Cassandra (which should be the same version, as it's defined on the parent pom). Basically, it's there as a module so that components can use its API. 

IIRC, we don't use the individual modules from Cassandra, only the -all artifact, so, that's the one that's added as a module.